src/main/java/org/launchcode/lifeorganizer/controllers/AuthenticationController.java

```diff
@@ -45,40 +47,75 @@ private static void setUserInSession(HttpSession session, User user) {
     @GetMapping("/signup")
     public String displaySignupForm(Model model) {
         model.addAttribute(new SignupFormDTO());
-        model.addAttribute("title","Sign-up");
+        model.addAttribute("title", SIGN_UP_TITLE);
```

Comment:
I think the constant is fine, since the user will be staying on the Sign up page while signing up.